framework/integration/test/basic/basic_test.go

```diff
@@ -62,9 +62,27 @@ func Test_Finalizer_Integration_Basic(t *testing.T) {
 		t.Fatal("expected", nil, "got", err)
 	}
 
-	// We directly pass the object to UpdateFunc.
-	operatorkitFramework.UpdateFunc(createdObj, createdObj)
+	// We update the object with a meaningless label to ensure a change in the
+	// ResourceVersion of the ConfigMap.
+	cm.SetLabels(
```

Comment:
Comments should explain this, the idea here is to change the `resourceVersion` and ensure we only add one finalizer.